src/sentry/discover/models.py

```diff
@@ -4,6 +4,7 @@
 from sentry.db.models.fields import JSONField
 
 MAX_KEY_TRANSACTIONS = 10
+MAX_TEAM_KEY_TRANSACTIONS = 100
```

Comment:
nit: this isn't being used anywhere yet

Reply:
Just adding that in since it'll be necessary soon 🙂